.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+### Thank you for your Pull Request! 
+
+We have developed a Pull Request template to aid you and our reviewers.  Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the xportr codebase remains robust and consistent.  
+
+### The scope of `{xportr}`
+
+`{xportr}`'s scope is to enable R users to write out submission compliant `xpt` files that can be delivered to a Health Authority or to downstream validation software programs.  We see labels, lengths, types, ordering and formats from a dataset specification object (SDTM and ADaM) as being our primary focus. We also see messaging and warnings to users around applying information from the specification file as a primary focus. Please make sure your Pull Request meets this **scope of {xportr}**. If your Pull Request moves beyond this scope, please get in touch with the `{xportr}` team on [slack](https://pharmaverse.slack.com/archives/C030EB2M4GM) or create an issue to discuss.
+
+Please check off each task box as an acknowledgment that you completed the task. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.
+
+### Changes Description
+
+_(descriptions of changes)_ 
+
+### Task List
+
+- [ ] The spirit of xportr is met in your Pull Request
+- [ ] Summary of changes filled out in the above Changes Description.  Can be removed or left blank if changes are minor/self-explanatory.
+- [ ] Check that your Pull Request is targeting the `devel` branch, Pull Requests to `main` should use the [Release Pull Request Template](https://github.com/pharmaverse/xportr/blob/main/.github/PULL_REQUEST_TEMPLATE/release.md)
+- [ ] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/).  Use `styler` package and functions to style files accordingly.
```

Comment:
Can we add a styler check with CI? I think that would be more robust then relying on the checkbox.

Reply:
I am down for a styler CI check.

Reply:
I made into issue #103

Reply:
I would like to leave in to help point out to newcomers on us following tidyverse style guide.

Reply:
PR #108 will add CI for styling. But I like the idea of having a checkbox here for newcomers. Maybe we can edit the last part to be a little more helpful. WDYT?
_"You can run the `styler::style_pkg()` to style the package."_